src/Appwrite/Network/Validator/URL.php

```diff
@@ -49,11 +49,17 @@ public function getDescription(): string
      */
     public function isValid($value): bool
     {
-        if (\filter_var($value, FILTER_VALIDATE_URL) === false) {
+        $sanitized_url = '';
+
+        foreach (str_split($value) as $character) {
+            $sanitized_url .= (ord($character) > 127) ? rawurlencode($character) : $character;
```

Comment:
I am not experienced with this, would it be please possible to include link to a resource that explains why we are looking for `ord() > 127`?

Reply:
refer [ doc](https://www.php.net/manual/en/function.ord.php)
@Meldiron